src/components/common/RichTextEditor.native.tsx

```
@@ -0,0 +1,116 @@
+  );
+
+  const handleParagraph = ({tintColor}: {tintColor: string}) => (
+    <Text style={{color: tintColor}}>N</Text>
+  );
+
+  const onImagePicked = async () => {
+    const result = await launchImageLibrary({
+      mediaType: 'photo',
+      selectionLimit: 1,
+    });
+    if (result && result?.assets && result?.assets[0]) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)